ci/matrix.xml

```diff
@@ -257,14 +241,13 @@
     sbt noDOM/run noDOM/testHtmlFastOpt noDOM/testHtmlFullOpt \
         withDOM/run withDOM/testHtmlFastOpt withDOM/testHtmlFullOpt \
         multiTestJS/test:run multiTestJS/testHtmlFastOpt multiTestJS/testHtmlFullOpt \
```

Comment:
Yes, it is intentional. We test the default a bit more heavily than the non-default, here.